docs/streams/tutorial.html

```diff
@@ -70,7 +70,7 @@ <h4 class="anchor-heading"><a id="tutorial_maven_setup" class="anchor-link"></a>
 
     <p>
         The <code>pom.xml</code> file included in the project already has the Streams dependency defined.
-        Note, that the generated <code>pom.xml</code> targets Java 8, and does not work with higher Java versions.
+        Note, that the generated <code>pom.xml</code> targets Java 11.
```

Comment:
@mjsax I assume we fixed this so it works with Java 11 or newer - is that correct?

Reply:
Yes we did update the archetype. Good catch.

Reply:
Shall I remove the Java version from here too?

Reply:
Good question. Not sure. Might be ok to leave for now 🤷